ツイートの画像の代替テキストを決めるためのフローチャートを追加 (#818)
kufu/smarthr-design-system: content/articles/operational-guideline/twitter.mdx

emojiを使ってもOKです。

### 画像の代替テキスト

必ず画像に代替テキストを設定しましょう。  
画像を直接見ないユーザーにも、画像が持つ情報を提供できます。  
代替テキストを設定しなければ、スクリーンリーダーや音声ブラウザ、テキストブラウザでは「画像」と読み上げられたり、表示されたりします。
これでは、晴眼者にしか意図が伝わりません。  

> 下図のフローチャートを参考に、代替テキストの内容を決めてください。

Reviewer: ```suggestion
代替テキストの内容は、以下の考え方に沿って決めてください。

- 画像に文字がある場合
	- 投稿本文に同じ内容の文字がある場合は、「装飾画像」と書きましょう
	- 投稿本文に同じ内容の文字がない場合は、画像に書かれている文字をそのまま書きましょう
- 画像に文字はないが、意味がある場合
	- 表やグラフなど複雑な情報の場合は、情報を要約して書くか、すべての情報を書きましょう
	- アイキャッチの場合は、画像の意味を書きましょう
- 画像に文字や意味がない場合
	- 「装飾画像」と書きましょう

下図は、上記内容のフローチャートです。
```

- 「Tweetの本文」という表現でしたが、ページ内に合わせて「投稿」という表現に変更しました。
- 内容が先で、それを補足するフローチャート、という順番にしました

> ![画像に文字があり、Tweetの本文に同じ内容の文字がある場合は、「装飾画像」と書きましょう。
> 画像に文字があり、Tweetの本文に同じ内容の文字がない場合は、画像に書かれている文字をそのまま書きましょう。
> 画像に文字はないが意味があり、表やグラフなど複雑な情報の場合は、情報を要約して書くか、すべての情報を書きましょう。
> 画像に文字はないが意味があり、表やグラフなど複雑な情報ではなく、アイキャッチである場合は、画像の意味を書きましょう。
> 画像に文字はないが意味があり、表やグラフなど複雑な情報でもアイキャッチでもない場合は、「装飾画像」と書きましょう。
> 画像に文字も意味もない場合は、「装飾画像」と書きましょう。](./images/flowchart_for_determining_the_alt_of_a_tweet.png)

Reviewer: ```suggestion
![代替テキストの内容を決定するフローチャート](./images/flowchart_for_determining_the_alt_of_a_tweet.png)
```
alt部分を「ほげのフローチャート」に変更しました

Author: よさそうです！コミットします！

emojiを使ってもOKです。

### 画像の代替テキスト

投稿に画像を添付する場合は、必ず画像に代替テキストを設定しましょう。  

> 画像を直接見ないユーザーにも、画像が持つ情報を提供できます。  
> 代替テキストを設定しなければ、スクリーンリーダーや音声ブラウザ、テキストブラウザでは「画像」と読み上げられたり、表示されたりします。
> これでは、晴眼者にしか意図が伝わりません。  

Reviewer: ```suggestion
代替テキストを設定することで、視覚障害のあるユーザーなど視覚情報に依存しない閲覧をするユーザーに限らず、視覚的に困難な状況のユーザーにも広く画像の意図を伝えることができます。
```

細かくてすみません＆元々もわりとそんなトーンだったのもありますが...

「これでは〜ません。」などがややネガティブ煽りが強めな印象を受けた（hogeしないとhogeになっちゃうよ、みたいな表現）のと、「こうするとこう良いことがあるんだよ〜」というポジティブな言い回しに調整しました。

表現は、以下の「アクセシビリティ > 画像の代替テキスト」から流用しています。
https://smarthr.design/accessibility/alternative-text/#h3-0

Author: 確かにポジティブな言い回しの方がメリットがわかりやすいし、パーソナリティにも沿っていてよさそうです！
コミットしますね〜！
